### PR TITLE
Region Module / Listener

### PR DESCRIPTION
**Pull request by itsjoekent**
Added region module, removed world protection

This --> It's currently missing player listener & the actual region check to see if a location is inside, but I wanted to push this first / get feedback <-- fixed in e475f5f

### CODE REVIEW

**Comment by itsjoekent** [comment]
Also sorry for commit spam.. :P

File: src/main/java/com/thevoxelbox/voxelguest/modules/worldprotection/WorldProtectionModule.java [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-import com.thevoxelbox.voxelguest.modules.GuestModule;
-import org.bukkit.World;
-
-import java.util.HashSet;
-import java.util.Set;
-
-/**
- * @author Monofraps
- */
-public class WorldProtectionModule extends GuestModule
```
Comment (itsjoekent) [inline src/main/java/com/thevoxelbox/voxelguest/modules/worldprotection/WorldProtectionModule.java:12]: @Monofraps Mike decided were going to just use regions rather then world based, don't think I'm trying to replace your code :P

Comment (nristock) [inline src/main/java/com/thevoxelbox/voxelguest/modules/worldprotection/WorldProtectionModule.java:12]: I HATE YOU! Just kidding - as I said it was just a test implementation .

Comment (itsjoekent) [inline src/main/java/com/thevoxelbox/voxelguest/modules/worldprotection/WorldProtectionModule.java:12]: OH NO, MONO DON'T HATE ME! PLEASE

Comment (nristock) [inline src/main/java/com/thevoxelbox/voxelguest/modules/worldprotection/WorldProtectionModule.java:12]: Whuhahaha! I gonna change the API when you're done coding the module to break EVERYTHING!

Comment (itsjoekent) [inline src/main/java/com/thevoxelbox/voxelguest/modules/worldprotection/WorldProtectionModule.java:12]: FUUUU

**Comment by itsjoekent** [comment]
Updated initial pull request, regions now have location checking support

**Comment by nristock** [comment]
Time is getting short.
I gonna fork yours, fix it and merge it back in.

**Pull request merged**